test/suites/upgrade/upgrade-block-2-minor.robot

```diff
@@ -61,7 +65,7 @@ Journal Should Have Information That MicroShift Skipped Restoring
     IF    ${version.minor} == 13    RETURN
 
     ${stdout}    ${rc}=    Execute Command
-    ...    journalctl --unit=microshift | grep "Restore skipped - data directory already matches backup to restore"
+    ...    journalctl --unit=microshift | grep "${EXPECTED_LOG}"
```

Comment:
The `--grep` option to journalctl would make this 1 command.

Reply:
it appears that `--grep`  in journalctl will ignore the log case, adding also `--case-sensitive=true`